README.md

```diff
@@ -72,7 +68,7 @@ In addition to our core research applications development team, we've had a numb
     PhD Candidate School of Anthropology</td>
     <td><a href="https://environment.arizona.edu/person/nancy-petersen"><img src="https://environment.arizona.edu/sites/environment.arizona.edu/files/styles/uaqs_medium/public/images/people/NancyP.jpg" width="25"> Nancy Petersen </a>
     <br />
-    AIRES Haury Program</td>
+    AIR Haury Program</td>
```

Comment:
Should we actually use the full name for the Haury program: "Agnese Nelms Haury Program in Environment & Social Justice"

Reply:
I agree with Thomas. As he is correct. We should use those words.